docs/using-the-compiler.rst

```diff
@@ -323,6 +323,8 @@ Input Description
         },
         // Metadata settings (optional)
         "metadata": {
+          // The metadata is appended at the end of the bytecode by default
+          "append": true,
```

Comment:
The `optimizer` section uses `enabled`. I also wonder how could these options reduced to a fewer, like an enum.